Realm/Realm/Logging/Logger.cs

```diff
@@ -120,12 +157,19 @@ protected Logger()
 
         internal static void LogDefault(LogLevel level, string message) => Default?.Log(level, message);
```

Comment:
This should be removed

Reply:
We currently have a bunch of references to this signature, and I think it could make sense to keep it in order to only have the `Logger` pass the `LogCategory.Realm.SDK` when logging from the SDK (instead of at all the calling sites).